datasets/modeled/datacenters/dataset.json5

```diff
@@ -0,0 +1,118 @@
+{
+  dataset_id: "decarb_2023_datacenters",
+  dataset_type: "modeled",
+  sector_description: "Residential and commercial",
+  description: "Datacenter load for the contiguous US for DECARB 2023 High Scenario, calculated using the EPRI 2024 whitepaper Analyzing Artificial Intelligence and Data Center Energy Consumption",
+  origin_creator: "Ashreeta Prasanna",
+  origin_organization: "NREL",
+  origin_contributors: [
+    "Ashreeta Prasanna",
+    "Elaine Hale",
+    "Sarah Smith"
+  ],
+  origin_project: "DECARB Year 1, Task 6",
+  origin_date: "2024-07-08",
+  origin_version: "v0.1.0",
+  data_source: "EPRI, LBL",
```

Comment:
Did the downscaling to states also come from EPRI, or somewhere else?

Reply:
Yes downscaling to states also from EPRI